stroyprombeton/tests/tests_selenium_admin.py

```diff
@@ -256,16 +256,22 @@ def test_tree_redirect_to_table_editor_page(self):
         test_search_value = self.browser.find_element_by_id('search-field').get_attribute('value')
         self.assertTrue(test_search_value)
 
-    def test_table_editor_search(self):
-        """Get first row from table for search test"""
-        first_product_name = self.browser.find_element_by_class_name('jqGrid_name')[0].text
-        search_input = self.browser.find_element_by_id('search_field')
-        search_input.clear()
-        search_input.send_keys(str(first_product_name) + Keys.ENTER)
-        wait(1)
+    def test_mark_search_on_table_editor(self):
+        """Search mark ЗПТ-100 on table editor"""
+        mark_name = 'ЗПТ-100'
+        self.send_keys_and_wait(mark_name, (By.ID, 'search_field'))
+
+        test_search_value = ''
+        first_row = self.browser.find_elements_by_class_name('jqgrow')[0]
+                        .find_elements_by_tag_name('td')
```

Comment:
we don't use intends with vary size (random whitespaces count).

Format like this instead:
```
first_row = (
    self.browser
    .find_elements_by_class_name('jqgrow')[0]
    .find_elements_by_tag_name('td')
)
```